CONTRIBUTING.md

```diff
@@ -15,6 +15,13 @@ This also applies to the code snippets in the markdown files.
 
 * Please wrap the code at 72 characters.
 
+## Comments
```

Comment:
@akutz Comments are for human to read, thus making it a real sentence makes sense to me. I prefer we have consistency among all types of files.